apps/cookbook/src/app/showcase/checkbox-showcase/checkbox-showcase.component.ts

```diff
@@ -54,4 +56,9 @@ Use the default attentionLevel 2 for checkbox lists.`,
       description: 'Emitted when the checkbox checked has changed',
     },
   ];
+
+  eventColumns: ShowcasePropertyColumns = {
+    Name: 'Name',
+    Description: 'Description',
+  };
```

Comment:
Consider adding `Signature` (`checkedChange` => `Promise<Boolean>`
```suggestion
    Type: 'Signature',
  };
```

Reply:
Not exactly sure what you mean, @jakobe

Reply:
Sorry, I can see it was not very clear 🙈 
I mean: Consider adding `Signature` column (here at this line) => and then add the signature to the `checkedChange` event definition in `ShowcaseProperty` array (with the value `Promise<Boolean>`

Reply:
Wouldn't that be an "extending cookbook content" issue on the edge of scope of this PR?

Reply:
Hmmm maybe, but in this particular case I would consider it good boyscouting :)
I.e. you are modifying the file anyway and stumble upon a tiny bit missing - the overhead of creating and managing a separate issue/PR compared with just-do-it isn't worth it IMHO... :)

Reply:
Totally agree regarding just-do-it 😊